app/src/main/java/com/example/concordia_campus_guide/Models/Routes/Subway.java

```diff
@@ -0,0 +1,22 @@
+package com.example.concordia_campus_guide.Models.Routes;
+
+import com.example.concordia_campus_guide.GoogleMapsServicesTools.GoogleMapsServicesModels.DirectionsStep;
+
+public class Subway extends TransportType {
+    private String color;
+
+    public Subway() {}
+
+    public Subway(DirectionsStep step) {
```

Comment:
We don't have duration for Subway? And Subway is the Metro right?

Reply:
Yes and yes

Reply:
We don't have duration b/c we're not interested in displaying the duration for the Subway in the routes overview section of the Routes Activity page.

Reply:
Alrighty then we don't need it sounds good